### ISSUE TITLE
Feature Inquiry: VAE Encoding Layer

### ISSUE THREAD

**Issue by asharron** [keras-team/keras#12811]
**Description**
In Variational Autoencoders (VAE) there is an encoding layer that is composed of the standard deviation and mean of the inputs in the coding space.

I looked through the documentation and couldn't find any pre-exisitng Keras layers that implemented this functionality found in standard VAEs.

I also couldn't find any issues that currently ask about this (though if there is, I apologize).

**Ask**
Is this feature being actively worked on? Or is the need so low that it isn't something included in the roadmap? 

Reason being is that I am writing my own Keras layer class to implement this functionality, and I wanted to know if I could open a pull request for this to be included in the Keras library if it does not exist. 

**TL;DR I want to make a pull request for Encoding Layers found in VAEs, but don't know if its already been done or currently worked on.**
